### ISSUE TITLE
Docs about theme are misleading

### ISSUE THREAD

**Issue by jenkin** [irfaan008/simple-skill-icons#1]
From the README.

> Some icons have a dark and light themed background. You can specify which theme you want as a url parameter. This is optional. **The default theme is dark.**

But here are all variants.

**No theme**

![My Skills](https://simpleskill.icons.workers.dev/svg?i=react,node.js,docker,mongodb,postgresql,awslambda,adobexd,angular)

**Dark theme**

![My Skills](https://simpleskill.icons.workers.dev/svg?i=react,node.js,docker,mongodb,postgresql,awslambda,adobexd,angular&theme=dark)

**Light theme**

![My Skills](https://simpleskill.icons.workers.dev/svg?i=react,node.js,docker,mongodb,postgresql,awslambda,adobexd,angular&theme=light)

So the default is colored icons, without any guarantee they works fine on dark or light backgrounds.

**Comment by irfaan008** [comment]
@jenkin You are right. This is true. I will update the doc accordingly. Also I am working on to put a rounded square background on each icon so if someone wants that pattern then that would be possible as well.

**Comment by jenkin** [comment]
Amazing! I suggest you introduce a new theme `framed` and let the default (no theme) to the raw icon with original colors.
